templates/build/small-h-card-detail.build.hbs

```diff
@@ -123,6 +123,30 @@
           \{{/if}}
         \{{/ifCond}}
 
+	      <!-- File template -->
```

Comment:
```suggestion
        <!-- File template -->
        \{{#ifCond type '==' 'file'}}
```

Reply:
Not addressed

Reply:
Not addressed

Reply:
Not addressed

Reply:
@tonytlwu I don't know how this happened but there are no extra spaces in my code editor to remove, should can I do that by commenting on your suggestion?
![image](https://user-images.githubusercontent.com/53430352/111177764-39c84b00-85b3-11eb-888b-c341e87d7487.png)

Reply:
@tonytlwu Fixed